package.json

```diff
@@ -32,8 +31,9 @@
     "eslint-plugin-prettier": "^4.0.0",
     "fastify": "^4.2.0",
     "fp-ts": "^2.11.2",
-    "jest": "^29.0.1",
     "prettier": "^2.4.1",
```

Comment:
We could remove it as well with `eslint-plugin-prettier`

Reply:
Yeah missed to remove the packages.